compiler-cli/src/new.rs

```diff
@@ -197,7 +222,8 @@ jobs:
 
     fn gleam_toml(&self) -> Result<()> {
         write(
-            self.root.join("gleam.toml"),
+            self.root
+                .join(FileToCreate::ConfigToml.location(&self.project_name)),
```

Comment:
If we're to go down the enum path these methods need to all be removed and replaced by iterating over the enum. The code would look something like this:

```rust
for file in FileToCreate::iter() {
    let path = directory_name.join(file.location(project));
    let contents = file.contents(project)
    write(path, file)?;
}
```

Reply:
Ah - I intentionally left those as-is, but can go ahead and move the content up and iterate here as well 🎉